### ISSUE TITLE
Settings / Team

### ISSUE THREAD

**Issue by MuhammadKhalilzadeh** [bluewave-labs/verifywise#376]
@gorkem-bwl 
There are details mentioned in the image that we need to think about if we want to proceed and implement them or not (I mean in V1).

What do you think?

![Screenshot (48)](https://github.com/user-attachments/assets/939ea9b5-7c71-4dfc-9e89-8d8665875dd5)

**Comment by gorkem-bwl** [comment]
As per the notes in the design,

1. **Organization name saving**: It's up to @HarshP4585 to define which table the organization name should be saved to.
2. **Modals on click**: There are no modals when user clicks on one of the rows. All the actions (e.g role change or user deletion) are done without clicking on the row itself and opening a modal.
3. **Role selection**: Everyone is administrator for v1.0 and there are no backend changes required for this area to work. We'll handle this after 1.0. 
4. **Data**: All data in this table should come from server side. 
5. **Save button**: We can remove the "Save" button as there is no need to take an action in this page. It would only save the role of the user, but we won't have this functionality for 1.0.

Invitations feature should be implemented for 1.0

**Comment by MuhammadKhalilzadeh** [comment]
Thank you @gorkem-bwl  for the details.
I'm gonna create a new issue based on this information and also the following image:

![Screenshot 2024-12-25 203151](https://github.com/user-attachments/assets/c24427fe-8756-4a34-a8b3-ca230ebbfce2)

**Comment by melpsh** [comment]
Does it have to be closed? I already have implemented this.

**Comment by gorkem-bwl** [comment]
> Does it have to be closed? I already have implemented this.

Yep, the issue owners need to close their issues after it's implemented.
